app/controllers/markets_controller.rb

```diff
@@ -0,0 +1,36 @@
+class MarketsController < ApplicationController
```

Comment:
Excellent! This is a wonderfully concise and straight-forward controller class. Nothing unnecessary, nothing confusing, just great :)